robocorp-code/src/robocorp_code/inspector/web/_web_inspector.py

```diff
@@ -386,14 +400,18 @@ def inject_picker(self, reason: str, page=None) -> bool:
                     log.debug("page is None in inject_picker!")
                     return False
 
-            log.debug(f"Picker was not injected (injecting now. Reason: {reason}).")
-            page.evaluate(_load_resource("inspector.js"))
+            # we need to wait for the load state before injecting otherwise the evaluation of the picker crashes
+            # not finding the body where to inject the picker in
+            page.wait_for_load_state()
+            time.sleep(0.2)
```

Comment:
This sleep is a bit troublesome (if we need a sleep it may mean that on different machines it may not be enough).

I don't want to block the PR due to that, but we should try to fix things so that it's not needed...